pkg/network/server.go

```diff
@@ -59,7 +59,7 @@ type (
 	Ledger interface {
 		extpool.Ledger
 		mempool.Feer
-		bqueue.Blockqueuer
+		bqueue.Blockqueuer[bqueue.Queueable]
```

Comment:
You need new interface for that. Starting from this commit `bqueue.Blockqueuer` serves only queue needs.